eng/common/generate-graphviz.ps1

```diff
@@ -0,0 +1,58 @@
+Param(
+  [string] $barToken,
+  [string] $gitHubPat,
+  [string] $azdoPat,
+  [string] $outputFolder,
+  [string] $darcVersion = '1.1.0-beta.19154.2',
+  [switch] $includeToolset
+)
+
+$ErrorActionPreference = "Stop"
+. $PSScriptRoot\tools.ps1
+
+function CheckExitCode ([string]$stage)
+{
+  $exitCode = $LASTEXITCODE
+  if ($exitCode  -ne 0) {
+    Write-Host "Something failed in stage: '$stage'. Check for errors above. Exiting now..."
+    ExitWithExitCode $exitCode
+  }
+}
+
+try {
+  Push-Location $PSScriptRoot
+    
+  Write-Host "Installing darc..."
+  . .\darc-init.ps1 -darcVersion $darcVersion
+  CheckExitCode "Running darc-init"
+
+  $DarcExe = "$env:USERPROFILE\.dotnet\tools"
+  $DarcExe = Resolve-Path $DarcExe
+  
+  if (!(Test-Path -Path $outputFolder)) {
+      Create-Directory $outputFolder
+  }
+    
+  $options = "get-dependency-graph --graphviz '$outputFolder\graphviz.txt' --github-pat $gitHubPat --azdev-pat $azdoPat --password $barToken"
```

Comment:
`$outputFolder\graphviz.txt` is used a couple of times, maybe make it a variable